onadata/libs/templates/account_lockout/lockout_start.txt

```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+{% blocktrans with lockout_time=lockout_time username=username %}
+Hi {{ username }},
+
+You have been locked out from your Ona account because of too many wrong passwords.
+Please wait for {{ lockout_time }} minutes before trying to access your account again.
```

Comment:
How long is the lock_out time? I thought it was a matter of hours/day not minutes.

Reply:
I think 30 minutes is a reasonable amount of time after 5 invalid password attempts.

Reply:
I think 30 minutes is a reasonable amount of time after 5 invalid login attempts.

Reply:
That sounds good to me too